### DISCUSSION TITLE
Frigate integration sensors / entities unavailable

### DISCUSSION THREAD

**Discussion by mikoday** [blakeblackshear/frigate#6229] [Ask A Question] [answered]
Please help me with my current setup. I'm unable to make any automations, because no sensors are working.

Frigate hass.io addon latest version: 0.12.0
Frigate integration for Home Assistant v4.0.0

http://ccab4aaf-frigate:5000 <- this url is not working
http://homeassistant.local:5000 <- so I used this

I see the preview picture of my camera:
http://homeassistant.local:8123/api/camera_proxy/camera.basement?token=[...]&width=80&height=80
camera.basement
106 × 80
image/jpeg

But unfortunately most sensors are unavailable (500: Internal Server Error):
http://homeassistant.local:8123/api/camera_proxy/camera.basement_person?token=[...]&width=80&height=80
http://homeassistant.local:8123/api/camera_proxy/camera.basement_cat?token=[...]&width=80&height=80

I see no issues in MQTT Explorer:
frigate_basement/basement/person
frigate_basement/basement/cat

**Comment by NickM-27** [comment, marked as answer]
you likely have not setup the mqtt integration in HA

**Reply by mikoday** [reply]
Yes, that was it. Thanks!
